core/platform/bulk_email/dev_mode_bulk_email_services_test.py

```diff
@@ -22,20 +22,24 @@
 from core.platform.bulk_email import dev_mode_bulk_email_services
 from core.tests import test_utils
 
+from typing import Any, Text # isort:skip # pylint: disable=unused-import
+
 
 class DevModeBulkEmailServicesUnitTests(test_utils.GenericTestBase):
     """Tests for mailchimp services."""
 
     def test_add_or_update_user_status(self):
+        # type: () -> None
         observed_log_messages = []
         def _mock_logging_function(msg, *args):
+            # type: (Text, *Any) -> None
             """Mocks logging.info()."""
             observed_log_messages.append(msg % args)
```

Comment:
@sajalasati Am I right here?

Reply:
Recommending the same types as defined in typeshed for the std lib method `logging.info` -  https://github.com/python/typeshed/blob/9af9cca7f3d9f4ea8507af4060a5adecb9525178/stdlib/logging/__init__.pyi#L76